src/main/java/org/mule/extension/api/runtime/MessageHandler.java

```diff
@@ -37,4 +37,11 @@
     void handle(MuleMessage<Payload, Attributes> message,
                                       CompletionHandler<MuleMessage<Payload, Attributes>, Exception> completionHandler);
 
+    /**
+     * Handles the {@code message} without notifying the result
```

Comment:
to this description, please add the following: "Useful to implement fire and forget use cases."